Solve wrong dependency name error

Found when running the github action is the following error:

```
[ERROR] dependency: org.melviz:melviz-runtime-shared:jar:1.0-SNAPSHOT (provided)
[ERROR] 	Could not find artifact org.melviz:melviz-runtime-shared:jar:1.0-SNAPSHOT
[ERROR] dependency: org.melviz:melviz-runtime-shared:jar:sources:1.0-SNAPSHOT (provided)
[ERROR] 	Could not find artifact org.melviz:melviz-runtime-shared:jar:sources:1.0-SNAP
```

--- a/melviz-bom/pom.xml
+++ b/melviz-bom/pom.xml
@@ -362,18 +362,18 @@
       <!-- Melviz Runtime -->
       <dependency>
         <groupId>org.melviz</groupId>
-        <artifactId>melviz-runtime-shared</artifactId>
+        <artifactId>melviz-webapp-shared</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.melviz</groupId>
-        <artifactId>melviz-runtime-shared</artifactId>
+        <artifactId>melviz-webapp-shared</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.melviz</groupId>
-        <artifactId>melviz-runtime-client</artifactId>
+        <artifactId>melviz-webapp</artifactId>
         <version>${project.version}</version>
         <scope>provided</scope>
         <type>war</type>

--- a/melviz-webapp-parent/melviz-webapp/pom.xml
+++ b/melviz-webapp-parent/melviz-webapp/pom.xml
@@ -38,7 +38,7 @@
     <dependencies>
       <dependency>
         <groupId>org.melviz</groupId>
-        <artifactId>melviz-runtime-shared</artifactId>
+        <artifactId>melviz-webapp-shared</artifactId>
         <scope>provided</scope>
         <version>${project.version}</version>
       </dependency>      
@@ -58,13 +58,13 @@
   <dependencies>
     <dependency>
       <groupId>org.melviz</groupId>
-      <artifactId>melviz-runtime-shared</artifactId>
+      <artifactId>melviz-webapp-shared</artifactId>
       <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>org.melviz</groupId>
-      <artifactId>melviz-runtime-shared</artifactId>
+      <artifactId>melviz-webapp-shared</artifactId>
       <scope>provided</scope>
       <classifier>sources</classifier>
     </dependency>

